batch.md

```diff
@@ -7,37 +7,51 @@ You may obtain a copy of the License at
 
     http://www.apache.org/licenses/LICENSE-2.0
 -->
-
+# Batch Connector
 The Flink connector library for Pravega makes it possible to use a Pravega stream as a data source and data sink in a batch program.  See the below sections for details.
 
 ## Table of Contents
-- [Data Source (InputFormat)](#data-source-inputformat)
-  - [Parameters](#parameters)
+- [FlinkPravegaInputFormat](#flinkpravegainputformat)
+  - [Pravega Batch Client](#pravega-batch-client)

```

Comment:
We don't need a section `Pravega Batch Client`

Reply:
Accepted suggestion